### PR TITLE
Fix resourceless tokens never playing their music.

### PR DESCRIPTION
**Pull request by marutypes**
**TLDR** This PR makes resourceless tokens (like many haunts) able to play their music.

## Motivation
I'm running PF2E campaigns, which often involve haunts. Haunts are proper tokens with initiative, actions, and sometimes even DCs. Unfortunately, they tend not to have HP or any other relevant resources. }

image

As such I've had to make this change locally in order to get haunts to play their music.

## Cause
When the module tries to figure out which playlist to play for a haunt in [Token.ts](https://github.com/elizeuangelo/fvtt-module-combat-music-master/blob/cc6140863d9ecae1a476136a940515c947f36469/module/token.ts#L151) we end up dividing by `attribute.max` and creating a NaN via division by zero. Since `NaN` is super weird, we can't ever get a true back from checking if it is greater or less than the current value. This means there is *no setting* that can cause the music to play, since NaN is neither less nor greater than any number.
image

## Reproduction
- Create or place a haunt. An example (from abomination vaults) looks like this
image
- Give it music using the tab this module adds
image
- Create a combat and add the token
image
- Begin combat
You will hear your default playlists play on the current main branch, and the token's music on mine.

### CODE REVIEW

**Comment by elizeuangelo** [comment]
I was thinking about fixing this some days ago! Thank you
